src/nvim/event/process.c

```diff
@@ -242,7 +241,7 @@ void process_stop(Process *proc) FUNC_ATTR_NONNULL_ALL
     case kProcessTypePty:
       // close all streams for pty processes to send SIGHUP to the process
```

Comment:
Not sure if this comment is still correct anymore, but the `process_close_streams` call is needed.